src/main/java/Bob.java

```diff
@@ -0,0 +1,119 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Bob {
+    public static void main(String[] args) {
+
+        displayWelcomeMessage();
+        List<Task> list = new ArrayList<>();
+
+        loop: while (true) {
+
+            Scanner in = new Scanner(System.in);
```

Comment:
scanner could be initialised outside the while loop so that it doesn't keep initialising